content/authorization/sign-in-with-livechat/index.mdx

```diff
@@ -66,268 +77,499 @@ The SDK is available in two formats: an NPM module and a CDN-hosted library.
+    }
+  }).catch((e)=>{
+    console.error("Failed to authorize user", e)
+  })
+};
+```
 
-```html
-<div class="livechat-login-button"></div>
+</CodeSample>
+
+During the creation of the Sign in LiveChat button, you can use [LiveChat Designs](https://livechat.design/).
+
+<Warning>
+
+[**Don't forget to test the give consent step**](/monetization/app-review-process/#common-problems). Use `prompt: "consent"` to force the app to **ask you for access** to certain resources. It’s necessary for you to test the app as if you were a user who installs the app from Marketplace.
+
+</Warning>
+
+# SDK documentation
+
+## Classes
+
+### **AccountsSDK**
+
+The main instance of the SDK used to authorize users in LiveChat Accounts.
+
+#### **constructor(options = {})**
+
+<CodeSample path={'Example class creation'}>
+
+```js
+const instance = new AccountsSDK({
+  client_id: '<your_app_client_id>'
+});
+```
+
+</CodeSample>
+
+##### options object
+
+The constructor accepts an `options` object with the following properties:
+
+| Property                             | Required | Description                                                                                                                                                  |
+| ------------------------------------ | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `client_id`                          | yes      | Application **Client Id** (can be found in <a href="https://developers.livechat.com/console/" rel="noopener nofollow" target="_blank">Developer Console</a>) |
+| `prompt`                             | no       | Use `consent` to force the consent prompt in a popup and the redirect flows. ([See the warning.](#step-3-prepare-button-container)). Default: `''`           |
+| `response_type`                      | no       | OAuth response type. Possible values: `token` or `code`. Default: `token`                                                                                    |
+| `popup_flow`                         | no       | `auto` - close the popup when not required; `manual` - always show the popup; default: `auto`                                                                |
+| `state`                              | no       | OAuth state parameter. Use it for better security.                                                                                                           |
+| `verify_state`                       | no       | Check if `state` matches after the redirect; default: `true`                                                                                                 |
+| `scope`                              | no       | Request exact scopes - must be configured for a given `client id`, a comma-separated string, default: `null`                                                 |
```

Comment:
The description seems unclear. Could we change it to:

Scopes needed by the application depending on what LiveChat API methods the app uses. Scopes must be configured for a given `client id`, a comma-separated string, default: `null`

Reply:
It's more like: `Scopes requested by current application` as with no scopes provides we are using default application scopes set.

Reply:
Hm, I'm not sure about it. How about?

> The scopes your application will request from the user?

Reply:
I'm also thinking about adding: `The scopes your application will request from the user. If not provided then all application scopes will be used`?

Reply:
I like that one :)  
I'd only change "used" to "requested"

> The scopes your application will request from the user. If not provided, then all application scopes will be requested.